lms/djangoapps/ccx/views.py

```diff
@@ -273,7 +271,7 @@ def create_ccx(request, course, ccx=None):
     )
 
     assign_staff_role_to_ccx(ccx_id, request.user, course.id)
-    is_course_licensing_enable = configuration_helpers.get_value('PCO_ENABLE_LICENSE_ENFORCEMENT', False)
+    is_course_licensing_enable = run_extension_point('PCO_ENABLE_COURSE_LICENSING')
```

Comment:
**PART 1:**
I have notice there are two approaches in terms of the usage of `'PCO_ENABLE_COURSE_LICENSING'`. In this part, first we checked `run_extension_point('PCO_ENABLE_COURSE_LICENSING')` assign it to a variable and then use it to call other run_extension_points. Therefore, those functions do not check if the plugin is enable. @anfbermudezme

Reply:
Besides that, Could you please assign this variable before you you can reuse it here? @anfbermudezme 

<img width="774" alt="image" src="https://user-images.githubusercontent.com/40271196/195398273-0913c3f6-67e8-4341-a23f-6f328ce3363c.png">